packages/components/src/components/data-table/_data-table-sort.scss

```diff
@@ -56,10 +58,19 @@
     opacity: 1;
   }
 
+  // Override for when `isSortable` is set on `DataTable` as well as `TableHeader`
+  .#{$prefix}--data-table.#{$prefix}--data-table--sort
+    th
+    > .#{$prefix}--table-header-label {
+    padding-right: $spacing-05;
+    padding-left: $spacing-05;
+  }
+
   // -------------------------------------
   // Th > Button > Span (span required for flex bugs in Safari)
   // -------------------------------------
-  .#{$prefix}--data-table--sort th .#{$prefix}--table-sort__flex {
+  .#{$prefix}--data-table--sort th .#{$prefix}--table-sort__flex,
+  th .#{$prefix}--table-sort__flex {
```

Comment:
is this extra selector intentional?

Reply:
It updates #7726, I put those styles in the `core` scss file, this PR moves them to the `sort` file with the preexisting selectors.

This selector is needed because in some instances you can pass `isSortable`  to the `TableHeader`, which does not add a global `.#{$prefix}--data-table--sort` class. Passing `isSortable` to `DataTable` gives us that class.

Reply:
I see so if L73 is needed then L72 would be redundant and we should be able to just simplify it down to `.#{$prefix}--table-sort__flex` right?

Reply:
Yeah most likely, let me just verify that the specificity is sufficient in all variants

Reply:
Updated 👍🏻